packages/devtools_app/test/shared/import_export_test.dart

```diff
@@ -10,6 +10,20 @@ import 'package:devtools_test/devtools_test.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() async {
+  test('Filename is sortable by time', () async {
```

Comment:
to test that these are sortable, maybe create few different file names with `generateFileName`, sort them, and then verify the sorting is as you'd expect

Reply:
Is a commit missing? I'm not seeing the change to this test to sort multiple filenames

Reply:
Oh whoops - disregard, I see it now.